Controllers/ToppingsController.js

```diff
@@ -5,9 +5,43 @@ const ObjectId = require("mongodb").ObjectId;
 
 const getAllToppings = (req, res) => {
   DbService.connectToDb(async (db) => {
-    const toppings = await ToppingsService.getAllToppings(db);
-    res.json(toppings);
+    const result = await ToppingsService.getAllToppings(db);
+    let jsonRes = ResponsesService.successful();
+    jsonRes.data = result;
+    return res.json(jsonRes);
+  });
+};
+
+const updateTopping = (req, res) => {
+  const data = {
+    id: ObjectId(req.body.id),
+    wins: req.body.wins,
+    battles: req.body.battles,
+  };
+  data.winPercent = Number(((100 * data.wins) / data.battles).toFixed(1));
```

Comment:
Why define this here instead of in the object above? Suggest just moving this line into it